tests/foreman/ui_airgun/test_oscappolicy.py

```diff
@@ -247,7 +247,7 @@ def test_positive_check_dashboard(session, module_host_group, module_loc,
         session.oscappolicy.create({
             'create_policy.name': name,
             'scap_content.scap_content_resource': oscap_content_title,
-            'scap_content.xccdf_profile': 'C2S for Red Hat Enterprise Linux 7',
+            'scap_content.xccdf_profile': 'C2S for Red Hat Enterprise Linux 6',
```

Comment:
are you sure that value present for `http://qe-sat64-rhel7***` machine?

Reply:
@oshtaier I have not tested that on http://qe-sat64-rhel7 as it's locked, used my provisioned sat for the tests, but for me it seems logical as soon as we are using in properties file , 
```
[oscap]
content_path= ... /oscap_data/ssg-rhel6-ds.xml
```

Reply:
test output on http://qe-sat64-rhel7*** machine
```
(robottelo) dlezz@dlezz:~/projects/robottelo-fork$ pytest -v tests/foreman/ui_airgun/test_oscappolicy.py::test_positive_check_dashboard
============================================ test session starts =============================================
platform linux -- Python 3.6.6, pytest-3.6.1, py-1.5.4, pluggy-0.6.0 -- /home/dlezz/.pyenv/versions/3.6.6/envs/robottelo/bin/python
cachedir: .pytest_cache
shared_function enabled - ON - scope: ffff - storage: file
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: wait-for-1.0.9, xdist-1.22.5, services-1.3.0, mock-1.10.0, forked-0.2, cov-2.5.1
collecting 1 item                                                                                            2018-10-11 15:21:50 - conftest - DEBUG - BZ deselect is disabled in settings

collected 1 item                                                                                             

tests/foreman/ui_airgun/test_oscappolicy.py::test_positive_check_dashboard PASSED                      [100%]

======================================== 1 passed in 1778.72 seconds =========================================
```